service/gradle/dependencies.gradle

```diff
@@ -37,6 +37,9 @@ dependencies {
     exclude group: "org.broadinstitute.dsde.workbench", module: "sam-client_2.12"
   }
 
+  //Terra Landing Zone Service
+  implementation group: 'bio.terra', name: 'terra-landing-zone-service', version: "0.0.1-SNAPSHOT"
```

Comment:
I saw `sam-client_2.12` sneak into the lockfile; we may need to exclude it here.

Or bite the bullet and do https://broadworkbench.atlassian.net/browse/TOAZ-181 :)

Reply:
I will exclude